addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1050,6 +1050,10 @@ function formatTables($editable) {
 export function getCSSRules(doc) {
     const cssRules = [];
     for (const sheet of doc.styleSheets) {
+        if (sheet.media?.mediaText === "print") {
```

Comment:
Hello! I fear this won't work if there is a rule that uses `@page` inside a query that targets both screen and print, for example `mediaText` could very well be `screen and (min-width: 480px), print`. Since at least some of those styles come from bootstrap or other libs, we don't control that css. I guess if such rule doesn't exist in our current version of Bootstrap then it's fine, but since this line will eventually reach master, maybe it will break again when we upgrade Bootstrap and they end up using a screen and print rule that also happens to use `@page`. I admit it's kind of a stretch, but I don't really see the benefit of this approach instead of explicitely excluding `@page` in `classToStyle` ?

Reply:
@dmo-odoo that's good point ! thank you for the feedback :pray: , I added `@page` to `SELECTOR_IGNORE` so that it will be skipped at `getCSSRules`

Reply:
Ah yes, that's way better, thanks! 👍

Reply:
@dmo-odoo does it look good to you ? so that we can r+ it :sweat_smile: